storybook/src/utils/AspectRatio/AspectRatio.stories.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { Paragraph } from '@amsterdam/design-system-react'
+import tokens from '@amsterdam/design-system-tokens/dist/index.json'
+import { Meta, StoryObj } from '@storybook/react'
+import { AspectRatio } from './AspectRatio'
+import type { AspectRatioProps } from './AspectRatio'
+
+const render = ({ aspectRatio }: AspectRatioProps) => (
+  <div className={`ams-aspect-ratio--${aspectRatio}`}>
+    <Paragraph
```

Comment:
The Paragraph as example is kind of confusing to me. I would expect an image since this is the only code example for aspect ratio and the most common usage is on an image. I would also like to know (personally) if it's ok to use the util to extend the `Image` class or that it still needs a wrapper.

Reply:
Changed to use a pink box.